recipe/meta.yaml

```diff
@@ -1,4 +1,3 @@
-{% set tensorflow_select_version = '1.0' if build_type == 'cpu' else '2.0' %}
```

Comment:
Was this line removed on purpose?

Reply:
yeah - I don't see a need to change this line

Reply:
This line isn't necessary here as `tensorflow_select_version` is already defined in the `config/conda_build_config.yaml`. So, I just removed it.

Reply:
Ah, OK